spec/presenters/curation_concerns/work_show_presenter_spec.rb

```diff
@@ -93,6 +93,20 @@
       end
     end
 
+    context "when there are >10 file sets" do
+      before do
+        12.times do
+          fs = create(:file_set)
```

Comment:
Sure, that would reduce the scope of the test and assume the indexing piece is tested sufficiently elsewhere.